Source/ActionController.swift

```diff
@@ -179,84 +179,85 @@ public class ActionController<ActionViewType: UICollectionViewCell, ActionDataTy
+        case .nibFile(let nibName, let bundle, _):
+            collectionView.register(UINib(nibName: nibName, bundle: bundle), forCellWithReuseIdentifier:ReusableViewIds.Cell.rawValue)
+        case .cellClass:
+            collectionView.register(ActionViewType.self, forCellWithReuseIdentifier:ReusableViewIds.Cell.rawValue)
         }
         
         // register main header
         if let headerSpec = headerSpec, let _ = headerData {
             switch headerSpec {
-            case .CellClass:
-                collectionView.registerClass(HeaderViewType.self, forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.Header.rawValue)
-            case .NibFile(let nibName, let bundle, _):
-                collectionView.registerNib(UINib(nibName: nibName, bundle: bundle), forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.Header.rawValue)
+            case .cellClass:
+                collectionView.register(HeaderViewType.self, forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.Header.rawValue)
+            case .nibFile(let nibName, let bundle, _):
+                collectionView.register(UINib(nibName: nibName, bundle: bundle), forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.Header.rawValue)
             }
         }
         
         // register section header
         if let headerSpec = sectionHeaderSpec {
             switch headerSpec {
-            case .CellClass:
-                collectionView.registerClass(SectionHeaderViewType.self, forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.SectionHeader.rawValue)
-            case .NibFile(let nibName, let bundle, _):
-                collectionView.registerNib(UINib(nibName: nibName, bundle: bundle), forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.SectionHeader.rawValue)
+            case .cellClass:
+                collectionView.register(SectionHeaderViewType.self, forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.SectionHeader.rawValue)
+            case .nibFile(let nibName, let bundle, _):
+                collectionView.register(UINib(nibName: nibName, bundle: bundle), forSupplementaryViewOfKind:UICollectionElementKindSectionHeader, withReuseIdentifier: ReusableViewIds.SectionHeader.rawValue)
             }
         }
         
         view.addSubview(collectionView)
         
         // calculate content Inset
         collectionView.layoutSubviews()
-        if let section = _sections.last where !settings.behavior.useDynamics {
+        if let section = _sections.last , !settings.behavior.useDynamics {
```

Comment:
This is caused by the Xcode migrator tool which replaced `where` with `,` (Notice that there is a whitespace before and after `where` but there shouldn't be white space there. I shouldn't have overlooked these. Sorry)